src/Drupal/UserTrait.php

```
@@ -154,6 +154,64 @@ public function userSetLastLoginTime(string $name, string $datetime): void {
+   * @code
+   * Given the following roles:
+   *   | name              | permissions                              |
+   *   | Content Editor    | access content, create article content   |
+   *   | Content Approver  | access content, edit any article content |
+   * @endcode
+   */
+  #[Given('the following roles:')]
+  public function userCreateRoles(TableNode $table): void {
+    foreach ($table->getHash() as $hash) {
+      if (!isset($hash['name'])) {
+        throw new \RuntimeException('Missing required column "name".');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '207', column '19'). (undefined)

(MissingImport)